### PR TITLE
Executes SCCP-347

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

File: tomls/omnibus-arbitrum-mainnet/perps/markets/btc.toml [hunk 1]
```diff
@@ -4,8 +4,8 @@ btc_perps_market_skew_scale = "<%= parseEther(String(35_000)) %>"
 btc_perps_market_max_funding_velocity = "<%= parseEther('9') %>"
 btc_perps_maker_fee_ratio = "<%= parseEther('0') %>"
 btc_perps_taker_fee_ratio = "<%= parseEther('0.0005') %>"
-btc_perps_max_market_size = "<%= parseEther(String(0)) %>"
-btc_perps_max_market_value = "<%= parseEther(String(0)) %>"
+btc_perps_max_market_size = "<%= String(1_200) %>"
```
Comment (noisekit) [inline tomls/omnibus-arbitrum-mainnet/perps/markets/btc.toml:7]: lost parseEther. Here and everywhere else

File: tomls/omnibus-arbitrum-mainnet/perps/markets/btc.toml [hunk 2]
```diff
@@ -4,8 +4,8 @@ btc_perps_market_skew_scale = "<%= parseEther(String(35_000)) %>"
 btc_perps_market_max_funding_velocity = "<%= parseEther('9') %>"
 btc_perps_maker_fee_ratio = "<%= parseEther('0') %>"
 btc_perps_taker_fee_ratio = "<%= parseEther('0.0005') %>"
-btc_perps_max_market_size = "<%= String(1_200) %>"
-btc_perps_max_market_value = "<%= String(50_000_000) %>"
+btc_perps_max_market_size = "<%= parseEther('1_200') %>"
+btc_perps_max_market_value = "<%= parseEther('50_000_000') %>"
```
[the remark this answers is not in the document]
Comment (kaleb-keny) [reply tomls/omnibus-arbitrum-mainnet/perps/markets/btc.toml:8]: sure sir 1 last update then

**Review by noisekit — APPROVED** [review]

**Review by Tburm — APPROVED** [review]
I ran fork tests using the SDK. Everything I tested here works as expected:
- All spot wrappers
- Successful USDC spot market swaps with low slippage (0.1%)
- Failed sETH and stBTC spot market swaps with low slippage (0.1%)
- Successful sETH and stBTC spot market swaps with high slippage (2%)
- Perps trading with USDx collateral
- Perps trading with sETH and stBTC collateral
- Liquidations using all collateral types, including checking the `RewardDistributed` events are as expected

**Comment by kaleb-keny** [comment]
hmmm @Tburm  what is the skew scale set on sUSDC 🤔  i do not recall setting a parameter for that one... there should be no slippage on sUSDC USDx swaps, just using pyth price

**Comment by kaleb-keny** [comment]
yeah it's not set, so there should be no slippage hopefully, only pyth price purely

**Comment by kaleb-keny** [comment]
for the  sETH and stBTC spot market swaps https://sips.synthetix.io/sips/sip-406/ i will present it to the new council hopefully

**Comment by Tburm** [comment]
@kaleb-keny What I meant was my tests were run with the slippage tolerance set to 0.1%, so it would pass if there was no slippage/fees. I could confirm for sure it is actually zero though.

**Pull request merged**